docs/en/RecordsInContexts.md

```diff
@@ -9,9 +9,9 @@ RiC is developed by International Council on Archives Expert Group on Archival D
 
 RiC has three components for now:
 
-- an __introduction to archival description (RiC-IAD)__ which is a brief introduction to the principles and purpose of archival description. __The latest version of RiC-IAD is RiC-IAD v0.2, dated December 2021, which can be downloaded [here](https://www.ica.org/sites/default/files/ric-iad-02_0_0.pdf)__.
-- a __conceptual model (RiC-Conceptual Model or RiC-CM)__ which defines the entities that exist in the world of archives, their attributes and the relations that can connect an entity to another one. A 0.1 version of RiC-CM was published in September 2016. __The latest version of RiC-CM is RiC-CM v0.2, dated July 2021, which can be downloaded [here](https://www.ica.org/sites/default/files/ric-cm-02_july2021_0.pdf)__.
-- an __OWL 2 ontology, (RiC-Ontology or RiC-O)__. It is a formal representation of Records in Contexts Conceptual Model (RiC-CM). It provides a generic vocabulary and formal rules for creating RDF datasets (or generating them from existing archival metadata) that describe in a consistent way any kind of archival record resource. It can support publishing RDF datasets as Linked Data, querying them using SPARQL, and making inferences using the logic of the ontology. __The latest public release of RiC-O, RiC-O v0.2, to which RiC-O Converter v2 is compliant, was published in February 2021__. It is fully documented in English and available here: [https://www.ica.org/standards/RiC/ontology](https://www.ica.org/standards/RiC/ontology).
+- an __introduction to archival description (RiC-FAD)__ which is a brief introduction to the principles and purpose of archival description. __The latest version of RiC-FAD is RiC-FAD v1.0, dated December 2023, which can be downloaded [here](https://www.ica.org/app/uploads/2023/12/RiC-FAD-1.0.pdf)__.
+- a __conceptual model (RiC-Conceptual Model or RiC-CM)__ which defines the entities that exist in the world of archives, their attributes and the relations that can connect an entity to another one. A 0.1 version of RiC-CM was published in September 2016. __The latest version of RiC-CM is RiC-CM v1.0, dated September 2023, which can be downloaded [here](https://www.ica.org/app/uploads/2024/01/ric-cm-1.0_0.pdf)__.
```

Comment:
"November 2023" instead of "September 2023"

Reply:
Also, "November 2023" instead of "December 2023" for RiC-FAD